frontend/app/templates/create-badges.hbs

```diff
@@ -144,13 +145,32 @@
     {{#each userError as |error|}}
     <p>{{error.message}}</p>
     {{/each}}
-    <div class="ui divider"></div>
     {{#if showProgress}}
-        <div class="ui segment">
-            <div class="ui centered aligned grid">{{progressState}}</div>
-            <div class="ui divider"></div>
-            {{ember-progress-bar progress=progress options=(hash color='orange')}}
+    <div class="ui segment">
+        <div class="ui centered aligned grid">{{progressState}}</div>
+        <div class="ui divider"></div>
+        {{ember-progress-bar progress=progress options=(hash color='orange')}}
+    </div>
+    {{/if}}
+    {{#if badgeGenerated}}
+    <div class="ui positive message">
+        <i class="close icon"></i>
+        <div class="header">
+            Badge generated successfully
         </div>
+        <p>Please visit this link to download badges
+            <a href="{{badgeGeneratedLink}}">Link</a>
+        </p>
+        <script type="text/javascript">
+            $('.message .close')
+                .on('click', function () {
+                    $(this)
+                        .closest('.message')
+                        .transition('fade')
+                        ;
+                });
+        </script>
+    </div>
```

Comment:
I can see under intended code here

Reply:
Using VSCode really, can't say it is a text editor though